config/environments/production.rb

```diff
@@ -46,7 +46,7 @@
   # config.action_cable.allowed_request_origins = [ "http://example.com", /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = ENV.key?('DISABLE_HTTPS') ? false : true
+  config.force_ssl = true
```

Comment:
Beware this will break running the app through docker-compose, as running in production will perform a redirect to https. It is why this ENV configuration exists, so in the docker-compose can be bypassed with `DISABLE_HTTPS: "1"`